.github/workflows/automation.yml

```
@@ -51,14 +51,21 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: scala-bionic
-      run: docker build . --file ./scala/Dockerfile.bionic --tag scala-bionic:$(date +%s)
-
+      run: docker build . --file ./scala/build/Dockerfile.bionic --tag "scala-bionic:$(date +%s)"
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)